src/thread_task.rs

```diff
@@ -346,91 +435,61 @@ unsafe fn create_filter_sbrow(
     } else {
         TaskType::ReconstructionProgress
     };
-    t.frame_idx = fc.index as c_uint;
-    Ok(task_idx)
+    let t = Rav1dTask {
+        type_0,
+        frame_idx: fc.index as c_uint,
+        tile_idx: 0,
+        sby: 0,
+        recon_progress: 1,
+        deblock_progress: 0,
+        deps_skip: AtomicI32::new(0),
+        next: Default::default(),
```

Comment:
```suggestion
        ..Default::default()
```

Since these fields weren't explicitly set previously, I think the intent is a bit clearer.